packages/storefront/src/pages/component-meta.md

```diff
@@ -1,48 +1,72 @@
 <template>
+  <div style="display: flex; justify-content: space-between; margin: 0 0 1rem">
+    <div>🚫 = deprecated<br>🛠 = breakpointCustomizable</div>
+    <p-switch @change="toggleProps" :checked="isToggled">Show all prop values</p-switch>
+  </div>
+
   <p-table>
     <p-table-head>
       <p-table-head-row v-html="headRow"></p-table-head-row>
     </p-table-head>
-    <p-table-body v-html="body"></p-table-body>
+    <p-table-body ref="body" @click="onClick" v-html="body"></p-table-body>
   </p-table>
 </template>
 
 <script lang="ts">
-import Vue from 'vue';  
+import Vue from 'vue';
 import Component from 'vue-class-component';
-import { getComponentMeta, TAG_NAMES } from '@porsche-design-system/shared';
-import type { ComponentMeta } from '@porsche-design-system/shared';
+import { TAG_NAMES } from '@porsche-design-system/shared';
+import { ComponentMeta, getComponentMeta} from '@porsche-design-system/component-meta';
 
 @Component
 export default class Code extends Vue {
   get headRow(): string {
-    return ['', ...TAG_NAMES].map((tagName) => `<p-table-head-cell>${tagName}</p-table-head-cell>`).join('');
+    return ['', ...TAG_NAMES].map((tagName) => {
+      const { isDeprecated } = getComponentMeta(tagName) || {};
+      return `<p-table-head-cell>${tagName}${isDeprecated ? '🚫' : ''}</p-table-head-cell>`;
```

Comment:
```suggestion
      return `<p-table-head-cell>${tagName}${isDeprecated ? ' 🚫' : ''}</p-table-head-cell>`;
```
I think, with spaces it looks better (cleaner):
<img width="834" alt="Bildschirm­foto 2023-03-10 um 10 16 00" src="https://user-images.githubusercontent.com/2981884/224275400-0ffe404d-fa3d-4e10-b5be-d1acafaa9596.png">